aioredis/commands/streams.py

```diff
@@ -3,13 +3,17 @@
 from aioredis.util import wait_convert, wait_make_dict, wait_ok
 
 
-def fields_to_dict(fields, type_=OrderedDict):
+def fields_to_dict(fields, type_=OrderedDict, encoding=None):
```

Comment:
Note to myself, would be good to move away from OrderedDict - that's completely unnecessary in Python 3.